_data/meltano/extractors/tap-messagebird/meltanolabs.yml

```diff
@@ -0,0 +1,53 @@
+capabilities:
+- catalog
+- state
+- discover
+- about
+- stream-maps
+- schema-flattening
+description: An omnichannel communications platform, built for global scale.
+domain_url: https://www.messagebird.com
+keywords:
+- meltano_sdk
```

Comment:
messagebird

Reply:
I decided to exclude this since messagebird is in the tap name so you already get a hit for messagebird when searching, is there another reason for adding this? Feel free to open another PR if so.